### PR TITLE
refactor: split up dev chain guide

### PR DESCRIPTION
**Pull request by wischli**
## fixes KILTProtocol/ticket#2005
* Splits up dev chain guide into three sections
  * Standalone: Build your own blockchain
  * Peregrine: Connect with test network + differences between Standalone and Parachain runtimes
  * Production: Recommendation to run full node + connect with Spiritnet
* Replaces previously existent Javascript codeblock with reference to the quick start
* The majority of the content was untouched and just moved from one file to the new one (please see below for the three bigger blocks of new content)

### CODE REVIEW

File: docs/develop/01_sdk/03_Chain setup for development/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+1. [Running your own KILT blockchain locally](./01_standalone-setup.md),
+2. [Connecting to the Peregrine test network](./02_peregrine-setup.md) and
+3. [Connecting to the Spiritnet production network](./03_prod-chain-setup.md).
+4. Moreover, we will explain the [differences between our _Standalone_ and _Parachain_ runtimes](./02_peregrine-setup.md#standalone-vs-parachain-peregrinespiritnet).
+
+## Set up your project
+
+We expect you to already have a small project which can connect and potentially interact with a blockchain given its WebSocket address.
+In case this puzzle piece is missing for you, please have a look at our [Quickstart](../01_quickstart.md#connecting-to-kilt-blockchain) which will provide you with all necessary means to create and run your script.
+
```
Comment (wischli) [inline docs/develop/01_sdk/03_Chain setup for development/index.md:26]: New content

File: docs/develop/01_sdk/03_Chain setup for development/03_prod-chain-setup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+In our [blockchain section](../../02_chain/01_introduction.md), you can find a [tutorial on how to run your own full node](../../02_chain/04_fullnode.md).
+
+## Connect with the network
+
+Simply replace the WebSocket address of your [your script](./index.md#set-up-your-project) or application with `wss://kilt-rpc.dwellir.com`.
+
+You can either use your own frontend or the [Polkadot JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkilt-rpc.dwellir.com/explorer) to interact with the chain.
+Moreover, you can use [Subscan](https://spiritnet.subscan.io/) as a chain explorer.
+For a full list of deployments and services, please have a look [here](../../02_chain/03_deployments.md).
```
Comment (wischli) [inline docs/develop/01_sdk/03_Chain setup for development/03_prod-chain-setup.md:24]: New content

File: docs/develop/01_sdk/03_Chain setup for development/02_peregrine-setup.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Since the native token of Peregrine, the _PILT_, does not have any economic value, you can request 100 PILT from the [Peregrine faucet](https://faucet.peregrine.kilt.io).
+
+## Connect with the network
+
+Simply replace the WebSocket address of your [your script](./index.md#set-up-your-project) or application with `wss://peregrine.kilt.io/parachain-public-ws`.
+
+You can either use your own frontend or the [Polkadot JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fperegrine-stg.kilt.io%2Fpara-public-ws#/explorer) to interact with the chain.
+Moreover, you can use [Subscan](https://kilt-testnet.subscan.io/) as a chain explorer.
+For a full list of deployments and services, please have a look [here](../../02_chain/03_deployments.md).
```
Comment (wischli) [inline docs/develop/01_sdk/03_Chain setup for development/02_peregrine-setup.md:22]: New content

**Review by tjwelde — APPROVED** [review]
I like it _very_ much!

**Pull request merged**